### ISSUE TITLE
[hue] Ambilight TV doesn't restore light values set by OpenHAB

### ISSUE THREAD

**Issue by maniac103** [eclipse/smarthome#4992]
I have an 'Ambilight+Hue' enabled TV which can use Hue lights as part of its Ambilight, controlling their color depending on the TV image contents. When turning off the TV, there's a difference in behavior of the lamp depending on the way I used for setting the color prior to turning on the TV:

- When I use the Hue app to control the color, the TV correctly restores that color when it's turned off
- When I use OpenHAB to control the color, the color is set to a light blue when turning off the TV

From OpenHAB code it's not obvious why that would happen, but one obvious difference between the Hue app and OpenHAB is that the former always sets the color as XY values, while OpenHAB uses HSB. I assume the TV firmware is only able to deal with (and, more importantly, was only tested with) XY values, as that's what the Hue app uses.

When #4959 is merged, it therefore might make sense to always write color updates as XY to match the Hue app behavior. I'll verify whether that fixes the issue in question.

**Comment by maniac103** [comment]
Color restoring indeed works fine when applying this patch: https://github.com/maniac103/smarthome/commit/24ac104c385cd98624031d1189d383ed79d5c292 ... I'll submit a PR once #4959 is merged.

**Comment by triller-telekom** [comment]
@maniac103 Since #4959 has been merged since a while, do you still want to come up with a PR?

**Comment by maniac103** [comment]
@triller-telekom Sure; I wasn't aware that #4959 is merged. I'll look into it as soon as I find some free time.

**Comment by triller-telekom** [comment]
@maniac103 Thanks, I am looking forward to see your PR!

**Comment by triller-telekom** [comment]
@maniac103 Is this still an issue? If so, would you like to come up with a PR? otherwise please close this issue.

**Comment by maniac103** [comment]
Yes, it is, and yes, I'd still like to come up with a PR, but I still haven't found time to do it :-(
